### ISSUE TITLE
Add a display indication of where keyframes are

### ISSUE THREAD

**Issue by eharris** [ozmartian/vidcutter#237]
There should be an indication of when the current position is a keyframe, and where/how far away the prior/next keyframes are.  This would greatly improve the ability to select the best cut point when SmartCut is not enabled.

It would also be great to have keyboard shortcuts to seek to prior and next keyframe so you can see exactly where the cut will occur (without SmartCut).

**Comment by frederickjh** [comment]
Agreed and as I mentioned in another issue adding a button to move the cut points to the key frames where it will actually but cut at would allow previewing a clip before saving it to a file.
